config/cmake/modules/FindCaliper.cmake

```
@@ -16,7 +16,28 @@
 
 include(MfemCmakeUtilities)
 mfem_find_package(Caliper CALIPER CALIPER_DIR
-       	"include" "caliper/cali.h"
-       	"lib" "caliper"
-  	"Paths to headers required by Caliper."
-       	"Libraries required by Caliper.")
+      "include" "caliper/cali.h"
+      "lib" "caliper"
+      "Paths to headers required by Caliper."
+      "Libraries required by Caliper.")
+
+# Append adiak path/lib if the user provided ADIAK_DIR
+if(ADIAK_DIR AND EXISTS ${ADIAK_DIR})
+    list(APPEND CALIPER_INCLUDE_DIRS ${ADIAK_DIR}/include)
+    
+    unset(_lib)
+    find_library(_lib "adiak" NO_CACHE PATHS ${ADIAK_DIR}/lib)
```

I think this should be:
```suggestion
    find_library("adiak" NO_DEFAULT_PATH PATHS ${ADIAK_DIR})
```

---

Good suggestion -- I was able to change the `adiak` import to `find_package` by playing with the `PATHS`.